dart/lib/src/sentry_options.dart

```diff
@@ -228,6 +232,16 @@ class SentryOptions {
   /// Whether to send personal identifiable information along with events
   bool sendDefaultPii = false;
 
+  /// Configures whether to record exceptions for failed requests.
+  /// Examples for captures exceptions are:
+  /// - In an browser environment this can be requests which fail because of CORS.
+  /// - In an mobile or desktop application this can be requests which failed
+  ///   because the connection was interrupted.
+  bool captureFailedRequests = false;
```

Comment:
Maybe this should be renamed to `captureFailedHttpRequests`. Since it's now a property on SentryOptions the http context is lost, so that may be worth adding back. Maybe also point out that the user has to use the SentryHttpClient or the dio integration for it do anything at all.